.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+on:
+  label:
+    types: [created]
```

Comment:
Label? Looks like this is going to be triggered on any label creation? Perhaps, there's a case for sticking some filtering on the job level?

Reply:
Yeah the environment has some filtering but I can actually do some filtering here too, good idea!

I just realized that I meant to get pushed tags, not labels. I must have been excessively tired when I wrote this originally...